### PR TITLE
Portfolio contact

### PR DESCRIPTION
**Pull request by mbonabucya**
##  Contact form description 

Add contact header 
Add contact description
Add contact form fields(name, email, textarea)
add get in touch button
Integrate formspree in the action of the form

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,18 +37,25 @@ Open index.html located at the route folder of the project
...
+if you feel like want to contribute , Please reach out to me via above the social media
 
-## Show your Support 
+## Show your Support
 
 Give a ⭐️ if you like this porto folio!
 
 ## Acknowledgments
+
 N/A
```
Comment (Kakalanp) [inline README.md:58]: [OPTIONAL]

Always try to look for some acknowledgments to add, it will make your project seem more humble :hugs:

**Review by mukhammadrizooff — APPROVED** [review]
Hi @mbonabucya , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
- All linters passed 💯 
- Correct use of Gitflow 💯 
- Descriptive messages 💯  
- Good implementation 💯
## Optional suggestions
- [ ] [OPTIONAL] I thought you were mistaken in choosing the color of the `Name` and `Email` input placeholders, please choose a slightly lighter color for it, such as `Message` textare.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 2]
```diff
@@ -526,3 +548,95 @@
...
+.my-form button:hover {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 12px;
+  width: 150px;
+  height: 48px;
+  background-color: #2230d2;
```
Comment (Kakalanp) [inline css/style.css:621]: [OPTIONAL]

It seems like your button wiggles when hovering and doesn't have the right color, the hex code should be `#6070FF`, make sure you're using the same margin/padding/border! :sweat_smile:

**Review by Kakalanp — APPROVED** [review]
# STATUS: APPROVED :green_circle: 

### Your project is approved @mbonabucya! :tada: 

There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :clinking_glasses: 

almost there

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**